src/modules/roles/services/role.application.service.spec.ts

```
@@ -18,6 +16,8 @@ import {
 import { RoleStatus, RoleType, SystemRoleKey } from '../interfaces/system-role-template.interface';
 import { RolePermissionRepository } from '../repositories/role-permission.repository';
 import { RoleRepository } from '../repositories/role.repository';
+import { RoleCacheService } from './role-cache.service';
```

ESLint

[error] `./role-cache.service` import should occur before import of `../../../enums`

(import/order)